### ISSUE TITLE
Disable XPDeprecationWarning

### ISSUE THREAD

**Issue by Margen67** [YaLTeR/OpenAG#65]
https://ci.appveyor.com/project/YaLTeR/openag/builds/29067902/job/u7cpcqh8xs5rmmue#L46
This will help bring down the warning count.

**Comment by YaLTeR** [comment]
Try this: https://stackoverflow.com/a/47436428

**Comment by Margen67** [comment]
I tried, but I'm pretty sure I'm doing it wrong: https://github.com/Margen67/OpenAG/commit/73bc234ba1fed0cf3efe75a124295d20cbdc3457

**Comment by YaLTeR** [comment]
Yes, you're doing it in the wrong place. You should do it in the main `CMakeLists.txt` after the `OpenAG` target is defined (with `add_library` IIRC).

**Comment by Margen67** [comment]
Still fails: https://github.com/Margen67/OpenAG/commit/1f2663a51e859a0ea22a60006e301cbcfde146b7

**Comment by YaLTeR** [comment]
You need to take the line
```cmake
set_target_properties(client PROPERTIES VS_GLOBAL_XPDeprecationWarning "False")
```
and put it somewhere after the line
```cmake
add_library(client SHARED ${SOURCE_FILES})
```

**Comment by Margen67** [comment]
It didn't fail, but didn't do much either: https://github.com/Margen67/OpenAG/commit/60656d24153a73dbfc5addd86c319cb7c262aad4

**Comment by YaLTeR** [comment]
Maybe it should be `false`

**Comment by Margen67** [comment]
Is it right at all? https://stackoverflow.com/questions/53841470/how-to-disable-warning-msb8051-support-for-targeting-windows-xp-is-deprecated/53965471#53965471
https://stackoverflow.com/a/47436428

**Comment by YaLTeR** [comment]
You should check which vcxproj ends up being generated and if the property is present there
